Rakefile

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env rake
+# frozen_string_literal: true
+
+require "json"
+
+task "default" => [:test]
+desc "Runs the tests!"
+
+task "test" do
+  def valid_json?(json)
+    JSON.parse(json)
+    true
+  rescue JSON::ParserError, TypeError => e
+    false
+  end
+
+  # test if _site/feed.json is valid
+  File.open("_site/feed.json", "r") do |f|
+    json = f.read
+
+    if valid_json?(json)
```

Comment:
We should do JSON schema test

Reply:
One step at a time.
Especially, as we don't have a schema.

Reply:
Now that we at least know that the [`json` can be parsed](https://github.com/ps-resources/es-offerings-site-feed/actions/runs/6234587768/job/16922082850), @igwejk do you want to start on a schema + its validation?

Reply:
yep @stoe... I wrote one as a suggestion. Let me know what you think.